beherabikram

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Sara Jaoude",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado"
+  "displayStatsPublic": true,
+  "bio": "I am a MERNStack developer",
```

Comment:
Duplicate **bio** tag. Consider removing this or the first one and combine the text 
```suggestion
  "bio": "I am a MERNStack developer. Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
```

Reply:
Given that the section `Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado"` is the Bio used in the example docs and my own personal bio I suggest leaving only `I am a MERNStack developer`